.github/labeler.yml

```diff
@@ -1,13 +1,31 @@
+infrastructure:
+  - .circleci/*
+  - .github/**/*
+
 dependencies:
   - poetry.lock
   - pyproject.toml
+  - mobileapp/package.json
+  - mobileapp/package-lock.json
+  - mockserver/package.json
+  - mockserver/yarn.lock
+  - web/package.json
+  - web/yarn.lock
 
 frontend 🎨:
-  - web/src/**/*
+  - any: ['web/**/*', '!web/public/locales/**/*']
 
 translations 🗣:
   - web/public/locales/**/*
 
+tests:
+  - tests/**/*
+  - parsers/test/**/*
+
+docker:
+  - docker-compose.yml
+  - web/Dockerfile
```

Comment:
These files are rarely updated and not worth it's own label - maybe put them under infrastructure? :)

Reply:
I thought it made sense split it out as it's another "language" as python and js but I can do that if you prefer.

Reply:
I was thinking it would be nice to clean up our labels and I think it's nicer to group things by "functionality" than language :)

Reply:
That makes sense. I'll push the changes soon.

It appears that dependabot already created a python label though, sorry about that. 😅

Reply:
Haha no worries 😄